acme/db/nosql/nosql.go

```diff
@@ -11,17 +11,18 @@ import (
 )
 
 var (
-	accountTable                        = []byte("acme_accounts")
-	accountByKeyIDTable                 = []byte("acme_keyID_accountID_index")
-	authzTable                          = []byte("acme_authzs")
-	challengeTable                      = []byte("acme_challenges")
-	nonceTable                          = []byte("nonces")
-	orderTable                          = []byte("acme_orders")
-	ordersByAccountIDTable              = []byte("acme_account_orders_index")
-	certTable                           = []byte("acme_certs")
-	certBySerialTable                   = []byte("acme_serial_certs_index")
-	externalAccountKeyTable             = []byte("acme_external_account_keys")
-	externalAccountKeysByReferenceTable = []byte("acme_external_account_key_reference_index")
+	accountTable                              = []byte("acme_accounts")
+	accountByKeyIDTable                       = []byte("acme_keyID_accountID_index")
+	authzTable                                = []byte("acme_authzs")
+	challengeTable                            = []byte("acme_challenges")
+	nonceTable                                = []byte("nonces")
+	orderTable                                = []byte("acme_orders")
+	ordersByAccountIDTable                    = []byte("acme_account_orders_index")
+	certTable                                 = []byte("acme_certs")
+	certBySerialTable                         = []byte("acme_serial_certs_index")
+	externalAccountKeyTable                   = []byte("acme_external_account_keys")
+	externalAccountKeysByReferenceTable       = []byte("acme_external_account_key_reference_index")
```

Comment:
it's an external_account_keyID by reference, right? You use keyID in the table and variable name below, but not in this one.

Reply:
Fixed in bf21319e76bf65b946ce5723de19f10059ee7f86.